customizations/generator/openapi/src/main/kotlin/com/expediagroup/sdk/generators/openapi/OpenApiSdkGenerator.kt

```diff
@@ -135,7 +135,7 @@ class OpenApiSdkGenerator {
                                 SupportingFile(
                                     "client.mustache",
                                     "$packagePath/client/",
-                                    "${namespace}Client.kt"
+                                    "${namespace.replaceFirstChar(Char::titlecase)}Client.kt"
```

Comment:
Maybe we should use the previously used `pascalCase` method since it's safer than an AdHoc approach?